crates/bevy_winit/Cargo.toml

```diff
@@ -23,6 +23,7 @@ bevy_utils = { path = "../bevy_utils", version = "0.5.0" }
 
 # other
 winit = { version = "0.25.0", default-features = false }
+approx = { version = "0.5.0", default-features = false }
```

Comment:
Do we *need* a new crate? Wouldn't silencing clippy, or using epsilon work?

Reply:
Maybe? We already depend on this transitively in the `bevy_text` crate, and I don't think it's that big. I can make the function inline though.

Reply:
ah, if the dep already exists it's probably fine

Reply:
This feels small enough that it might be better to include it in bevy_core. We already have FloatOrd, which is in a similar vein. For now this is fine though.